src/internal/__tests__/curry.test.js

```diff
@@ -41,7 +41,52 @@ describe('variadicCurryWithValidation', function () {
     const f = variadicCurryWithValidation((x) => x === null, 'null', (x) => 0, func, false)
     expect(() => f(1)(2)(3)).toThrowError(new Error('func takes up to 2 arguments, followed by null. You already passed 3 arguments and the last argument was not null'))
   })
-  it('works on functions which take a variable number of iterables', function () {
 
+  describe('explicit partial application', function () {
+    let curriedMethod
+
+    beforeEach(function () {
```

Comment:
Instead of `let`/`beforeEach`, you can use a factory function (function that returns `curriedMethod`)

```javascript
const makeCurriedMethod = variadicCurryWithValidation(/* ... */)

it('curried methods have a partial method', function () {
  expect(typeof makeCurriedMethod()).toHaveProperty(expect.any(Function))
})
```